Source/Core/TextureResource.cpp

```diff
@@ -137,7 +137,7 @@ bool TextureResource::Load(RenderInterface* render_interface) const
 			int layout_id;
 			int texture_id;
 
-			if (sscanf(source.CString(), "?font::%p/%p/%d/%d", &handle, &layer_id, &layout_id, &texture_id) == 4)
+			if (sscanf(source.CString(), "?font::%p/%p/%d/%d", (void**)&handle, (void**)&layer_id, &layout_id, &texture_id) == 4)
```

Comment:
This seems like no less of an undefined behavior than before.  Should read to a void* variable and then cast it.

Reply:
That's not undefined behaviour but _incorrect type_ (`warning: format ‘%p’ expects argument of type ‘void**’, but argument 3 has type ‘Rocket::Core::FontFaceHandle**’`).

Reply:
Yeah so I'm saying, have something like `void* handle; scanf(..., &handle);`

Reply:
That was my first idea when I was trying to fix that but then I thought that the type is correct and it is `scanf` and `printf` that are not supporting typed pointers so I went with the cast. At the end of the day I think it's safer to work with typed pointers and just convert/cast them when a function doesn't provide support for that  than having void pointers all around.
Does that make sense?

Reply:
I believe it's an undefined behavior to write a T* through void**. So you have to have a void* variable to actually fix the issue.